packages/threat-composer/src/components/workspaces/WorkspaceInsights/components/Overview/index.tsx

```diff
@@ -87,6 +94,22 @@ const Overview: FC<WorkspaceInsightsProps> = ({
     return filterThreatsByMetadata(statementList, 'Priority').length;
   }, [statementList]);
 
+  const [completedMitigations, totalMitigations] = useMemo(() => {
+    return [
+      mitigationList.filter((m) => m.status === MITIGATION_STATUS_RESOLVED
+        || m.status === MITIGATION_STATUS_RESOLVED_ABANDONED).length,
```

Comment:
Upon reflection, 'Abandoned' carries a negative connotation - and that it is simply left behind or neglected. I propose we change to 'Will not action' which carries slightly less negative connotation in my opinion, and is clear that it's a _decision_:

`MITIGATION_STATUS_RESOLVED_WILLNOTACTION`